src/main/java/mat/client/shared/MeasureLibraryGridToolbar.java

```diff
@@ -19,6 +21,10 @@
     private Button shareButton;
     private Button cloneButton;
     private Button exportButton;
+    private Button fhirValidationButton;
+    private float version;
+
+    private static final float RUN_FHIR_VALIDATION_VERSION = 5.8F;
```

Comment:
1. BigDecimal is a better option.
2. The toolbar should not be aware of validation eligibility details